Pass meminfo & mmap information to Xen

Our version of Xen 4.7 requires this information to be passed despite
having boot services enabled.  This patch can be removed once we move to
Xen 4.9.

diff --git a/grub-core/loader/multiboot_mbi2.c b/grub-core/loader/multiboot_mbi2.c
index b0679a9..52b9a93 100644
--- a/grub-core/loader/multiboot_mbi2.c
+++ b/grub-core/loader/multiboot_mbi2.c
@@ -837,7 +837,7 @@ grub_multiboot_make_mbi (grub_uint32_t *target)
       }
   }
 
-  if (!keep_bs)
+  if (1 /* !keep_bs */)
     {
       struct multiboot_tag_mmap *tag = (struct multiboot_tag_mmap *) ptrorig;
       grub_fill_multiboot_mmap (tag);
@@ -859,7 +859,7 @@ grub_multiboot_make_mbi (grub_uint32_t *target)
       / sizeof (grub_properly_aligned_t);
   }
 
-  if (!keep_bs)
+  if (1 /*!keep_bs */)
     {
       struct multiboot_tag_basic_meminfo *tag
 	= (struct multiboot_tag_basic_meminfo *) ptrorig;
